frontend/src/scenes/session-recordings/player/Seekbar.tsx

```diff
@@ -54,6 +54,19 @@ export function Seekbar(): JSX.Element {
     return (
         <div className="rrweb-controller-slider">
             <div className="slider" ref={sliderRef} onMouseDown={handleDown} onTouchStart={handleDown}>
+                <div className="inactivity-bar">
+                    {sessionPlayerData?.metadata?.segments?.map((segment: RecordingSegment) => (
+                        <div
+                            key={`${segment.windowId}-${segment.startTimeEpochMs}`}
+                            className={clsx('activity-section', !segment.isActive && 'inactive-section')}
+                            style={{
+                                width: `${
+                                    (100 * segment.durationMs) / sessionPlayerData.metadata.recordingDurationMs
```

Comment:
Will this `/ sessionPlayerData.metadata.recordingDurationMs` properly take into account the multiple windows? Doesn't that add more stuff to the bar that the nr of seconds?

Reply:
Good question, `recordingDurationMs` is calculated based on the segments not just subtracting the start time from the end time, so it accounts for segments that overlap and when segments aren't continuous.